docs/user-guide/configure-zowe-runtime.md

```diff
@@ -609,24 +686,32 @@ The manual installation consists of the following steps.
       1. Define the BPX Resource and access for IZUSVR.
            ```
            SET RESOURCE(FAC)
+           ```
+           ```
            RECKEY BPX ADD(SERVER ROLE(IZUSVR) SERVICE(UPDATE) ALLOW)
+           ```
+           ```
            RECKEY BPX ADD(DAEMON ROLE(IZUSVR) SERVICE(UPDATE) ALLOW)
+           ```
+           ```
            F ACF2,REBUILD(FAC)
            ```
       1. Issue the following commands and review the output to check whether permission has been successfully granted:
            ```
            SET RESOURCE(FAC)
+           ```
+           ```
            LIST BPX
            ```
       </details>
 
-### Scripted install of the Zowe Cross Memory Server
+### Installing the Cross Memory Server using the script
 
-For users who have sufficient authority under their user ID on the z/OS instance where they are installing the Zowe cross memory server, a convenience script is provided in `xmem-server/zowe-install-apf-server.sh`.  If this script does not exist review the section [Creating the xmem-server/zss directory](#Creating-the-xmem-server/zss-directory)
+For users who have sufficient authority under their user ID on the z/OS instance where they are installing the Cross Memory Server, a convenience script is provided in `xmem-server/zowe-install-apf-server.sh`. If this script does not exist, review the section [Creating the xmem-server/zss directory](#Creating-the-xmem-server/zss-directory).
```

Comment:
Broken link: `[Creating the xmem-server/zss directory](#Creating-the-xmem-server/zss-directory)` should be `[Creating the xmem-server/zss directory](#creating-the-xmem-server-zss-directory)`

Reply:
Done